### DISCUSSION TITLE
[Feature] A way to select screenshots and hide them from timeline.

### DISCUSSION THREAD

**Discussion by chengcheng300** [immich-app/immich#8145] [Feature Request]
### The feature

I have many screenshots and downloaded images in my photos, I think the way to distingush them is by searching where camera info is "(blank)". However, there is currently no way to search by "(blank)".

I hope to find all my screenshots and put them into an Screenshot albumn, meanwhile, hide them from the timeline, because since they don't have camera infomation, many importing and exporting tools often use the exporting/importing date as their creation date which leaves a huge cluster of screenshots in the timeline. 

What like to hear your thoughts on this feature request. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
You can try search for "screenshot"

**Comment by ddwag1** [comment]
It would be nice to have a toggle to automatically hide screenshots from the timeline. Currently have to re do occurring context searches and manually archive. I guess having a smart album of screenshots then hiding them from the timeline would also work..?

**Comment by dominiwe** [comment]
I'd be interested in this too. The best way would be to be able to make a "smart album" that contains all screenshots based on context and then to be able to configure the global timeline to not display the set of images in that album.

**Comment by brewing8309** [comment]
would be great!

**Comment by thomasp001** [comment]
I'll second this. It would be good to not have screenshots appear in the memories.

**Comment by GreaseHeadD** [comment]
I have made a modified version of Salvoxia's [immich-folder-album-creator](https://github.com/Salvoxia/immich-folder-album-creator) which can detect and add screenshots to an album, and if desired hide the screenshots from the timeline by archiving them. You can find it at [immich-screenshots-organizer](https://github.com/GreaseHeadD/immich-screenshots-organizer)

**Comment by bo0tzz** [comment]
#3477

**Comment by qkrrmsdud1** [comment]
iOS labels screenshots in its filesystem. Is there a way for this info to be uploaded with photos to Immich?

**Reply by noidwasavailable** [reply]
Seems like iOS-labelled screenshots are put in description. Try Filters -> Search Type: Description and search for "Screenshot"
